tests/ci/docker_images/linux-x86/amazonlinux-2_clang-11x/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
what's this dockerfile used for?

Reply:
As a layer that has clang installed/configured for the cryptofuzz container. In LC, they use it for other stuff but if we are inclined we can flatten the hierarchy a bit to avoid this. Layering is best practice within the Docker ecosystem though.

Reply:
no objections to layering, but it looks like  `tests/ci/docker_images/linux-x86/amazonlinux-2_clang-11x_cryptofuzz/Dockerfile` doesn't depend on this or images produced by it, instead referencing `amazonlinux-2023:clang-15x`

Reply:
(no need to address until next PR in the series)

Reply:
This PR is direct copy of the source material (with some file renaming to make things diff better). The next PR updates the references.